packages/embark-profiler/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "embark-profiler",
+  "version": "4.0.0",
```

Comment:
Since it's a new package, I think the starting version should match the `lerna.json` master version number that we'll (probably) cherry-pick from `4.0.x` branch. Not sure if merge of this PR should hold off until we've done that, but it's probably a good idea. If we go ahead and merge this one, should remember to make another PR to bump it to match `lerna.json` when the time comes.